src/aliceVision/image/io.hpp

```diff
@@ -206,6 +206,8 @@ struct ImageReadOptions
     bool doWBAfterDemosaicing;
     std::string demosaicingAlgo;
     int highlightMode;
+    bool rawAutoBright;
+    float rawExposureAdjustment;
```

Comment:
```suggestion
    bool rawAutoBright = false;
    float rawExposureAdjustment = 1.0f;
```

Reply:
No need for that, initialization already done line 198.

Reply:
IMO it's more readable to write default values on class members directly.